src/components/RulesDialog.vue

```diff
@@ -1,239 +1,216 @@
 <template>
-  <v-dialog v-model="adaptedShow">
-    <template v-if="!hideActivator" #activator="{ props }">
-      <span v-bind="props">
-        <slot name="activator">
-          <v-btn
-            class="rules-button"
-            color="primary"
-            variant="outlined"
-            :size="buttonSize"
-          >
-            Rules
-          </v-btn>
-        </slot>
-      </span>
+  <base-dialog max-width="1250">
```

Comment:
Does the max-width assignment work here? I'd be surprised if it did anything bc I don't think BaseDialog pays attention to it

Reply:
This makes it bigger, without it it's around 650px max